src/ros_filter_bias_estimator.cpp

```diff
@@ -46,7 +46,7 @@ void RosFilterBiasEstimator::reset() {
     mtx_.lock();
     uncorrected_state_received_s_ = 0.0;
     uncorrected_speed_ = 0.0;
-    for(uint8_t counter = 0; counter < ESTIMATION_AXES; counter++)
+    for(uint8_t counter = 0; counter < 3; counter++)
```

Comment:
hardcoded 3?
is it supposed to be updated to `BIAS_ESTIMATOR_ESTIMATION_AXES` ?

Reply:
Apparently my computer ignored my cntrl+s. It's now fixed.